src/clr-icons/clr-icons-element.ts

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { ClarityIconsApi } from './clr-icons-api';
+import { ShapeTemplateObserver } from './utils/shape-template-observer';
+
+/* CLR-ICON CUSTOM ELEMENT */
+
+let clrIconId = 0;
+const offScreenSpan = document.createElement('span');
```

Comment:
Weird considering that this `document` has no guard but it was working?

Reply:
Strange yes, but the contents of this file do no make it into the SSR bundles.